docs/user-defined-redirects.rst

```diff
@@ -12,10 +12,19 @@ Quick Summary
 * From the left navigation menu, select :guilabel:`Redirects`.
 * In the form box "Redirect Type" select the type of redirect you want. See below for detail.
 * Depending on the redirect type you select, enter FROM and/or TO URL as needed.
-* When finished, click the SUBMIT Button.
+* When finished, click the ADD button.
 
 Your redirects will be effective immediately.
 
+.. note::
+
+    Since we serve documentation in a highly available way,
+    we do not run any logic when we're serving documentation.
+    This means that redirects will only happen in the case of a *404 File Not Found*.
+
+    In the future we might implement redirect logic in Javascript,
+    but this first version is only implemented in the 404 handlers.
```

Comment:
this is a little out of date, we run some logic when serving docs now. I think we can just mention that redirects only work on 404 (and maybe leave the second paragraph that mentions the js solution too).

https://github.com/readthedocs/readthedocs.org/blob/c857c6c3aa6c40ae616f30ff7ed587468ed285db/readthedocs/proxito/views/serve.py#L126-L135

I think at this point we only need to make the decision on how to implement them and how to make it in a way that doesn't introduce a lot of queries to the db.